phpstan-bootstrap.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+define('MAUTIC_ENV', '');
```

Comment:
@lenonleite there will be an upcoming change to `phpstan`, this change will be impacted with your work, so I thought of letting you know

Reply:
@mabumusa1 did you tag the wrong person?

Reply:
No, @lenonleite is working PR #12206, there is a lot going on with that PR regarding `.env` so I thought of giving him heads up about this change